crates/zed/src/zed/linux_prompts.rs

```diff
@@ -28,10 +30,18 @@ pub fn fallback_prompt_renderer(
         |cx| FallbackPromptRenderer {
             _level: level,
             message: message.to_string(),
-            detail: detail.map(ToString::to_string),
             actions: actions.iter().map(ToString::to_string).collect(),
             focus: cx.focus_handle(),
             active_action_id: 0,
+            detail: detail.filter(|text| !text.is_empty()).map(|text| {
+                cx.new_view(|cx| {
+                    let mut editor = Editor::auto_height(3, cx);
```

Comment:
Rather than using an Editor, could we use a Markdown element? That's how we implement the selections in the hover popover: https://github.com/zed-industries/zed/blob/aca23da9714e0482be8bee1b56695ac3ff1d4cc6/crates/editor/src/hover_popover.rs#L779